package-lock.json

```diff
@@ -1,7513 +0,0 @@
-{
```

Comment:
Are you sure we want to delete this?  I thought it was meant to be under version control.

Reply:
This would presumably allow you to have repeatable builds for any point in time.

Reply:
I noticed that it was rebuilt on first npm, and broom does a clean install each time.

Reply:
re-added from dev branch